tetrateci/README.md

```diff
@@ -59,3 +59,22 @@ Workflows of this repo are dependent on the following github secrets:
 13. TEST_HUB_PASS
 14. BINTRAY_ARCHIVE_API : [ Note : The archive API url, example - https://api.bintray.com/content/hellozee/istio-archives ]
 15. AZURE_RESOURCE : [ Note : The azure resource name corresponding to the creds ]
+
+#### Debugging
+If the workflow fails, the logs should be pretty clear why it failed. If still the problem cant't be debugged, the best suggestion would be to add the following workflow and ssh into the runner to run the steps manually.
```

Comment:
If this can be added as a first job to any workflow, can you rephrase the text to reflect that?